src/main/java/com/qaprosoft/carina/demo/enums/GenderEnum.java

```diff
@@ -0,0 +1,14 @@
+package com.qaprosoft.carina.demo.enums;
+
+import com.itextpdf.text.pdf.RadioCheckField;
+
+public enum GenderEnum {
+    MALE("male"),
+    FEMALE("female");
+
+    public static RadioCheckField getGenderEnum;
+
+    GenderEnum(String gender) {
+    }
+
```

Comment:
Add getter in enum:

 private final String value;

    GenderEnum(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }